config/initializers/new_framework_defaults.rb

```diff
@@ -0,0 +1,23 @@
+# Be sure to restart your server when you modify this file.
+#
+# This file contains migration options to ease your Rails 5.0 upgrade.
+#
+# Once upgraded flip defaults one by one to migrate to the new default.
+#
+# Read the Rails 5.0 release notes for more info on each option.
+
+# Enable per-form CSRF tokens. Previous versions had false.
+Rails.application.config.action_controller.per_form_csrf_tokens = false
```

Comment:
The intention with this file is to suppress the new defaults in Rails 5 until we have confirmed that we can safely apply them. By committing this file it implies we are unable to apply the new defaults?